### PR TITLE
SUS-2180 remove iw_api column

### PR DESCRIPTION
**Pull request by sqreek**
https://wikia-inc.atlassian.net/browse/SUS-2180
@Grunny @macbre @mixth-sense

### CODE REVIEW

File: includes/installer/DatabaseInstaller.php [hunk 1]
```diff
@@ -580,7 +580,7 @@ public function populateInterwikiTable() {
 			if ( $row == "" ) continue;
```
Comment (WikiaTech) [inline includes/installer/DatabaseInstaller.php:580]: MAJOR Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
MINOR 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 51 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 30 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 16 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [parserTest.inc#L696](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/tests/parser/parserTest.inc#L696): Remove this use of the "eval" function. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1523)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L32](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/api/ApiQuerySiteinfo.php#L32): Class "ApiQuerySiteinfo" has 29 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L38](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/api/ApiQuerySiteinfo.php#L38): The Cyclomatic Complexity of this function "execute" is 23 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L330](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/api/ApiQuerySiteinfo.php#L330): Remove this unused "$host" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L514](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/api/ApiQuerySiteinfo.php#L514): Do not use the global wgParser Parser instance. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L522](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/api/ApiQuerySiteinfo.php#L522): Do not use the global wgParser Parser instance. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L15](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/installer/DatabaseInstaller.php#L15): Class "DatabaseInstaller" has 39 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L318](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/installer/DatabaseInstaller.php#L318): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L489](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/installer/DatabaseInstaller.php#L489): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L513](https://github.com/Wikia/app/blob/73b543164387cf64ff916cfd7bccd37159586679/includes/installer/DatabaseInstaller.php#L513): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: includes/interwiki/Interwiki.php [hunk 2]
```diff
@@ -16,12 +16,11 @@ class Interwiki {
 	protected static $smCache = array();
 	const CACHE_LIMIT = 100; // 0 means unlimited, any other value is max number of entries.
```
Comment (WikiaTech) [inline includes/interwiki/Interwiki.php:17]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 51 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 30 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 16 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [parserTest.inc#L696](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/tests/parser/parserTest.inc#L696): Remove this use of the "eval" function. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1523)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L32](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/api/ApiQuerySiteinfo.php#L32): Class "ApiQuerySiteinfo" has 29 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L38](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/api/ApiQuerySiteinfo.php#L38): The Cyclomatic Complexity of this function "execute" is 23 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L330](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/api/ApiQuerySiteinfo.php#L330): Remove this unused "$host" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L514](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/api/ApiQuerySiteinfo.php#L514): Do not use the global wgParser Parser instance. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L522](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/api/ApiQuerySiteinfo.php#L522): Do not use the global wgParser Parser instance. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L15](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/installer/DatabaseInstaller.php#L15): Class "DatabaseInstaller" has 39 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L318](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/installer/DatabaseInstaller.php#L318): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L489](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/installer/DatabaseInstaller.php#L489): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L513](https://github.com/Wikia/app/blob/44066c78d84b0415c86e23b96bccd634aa7f2757/includes/installer/DatabaseInstaller.php#L513): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 51 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 30 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 16 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [parserTest.inc#L696](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/tests/parser/parserTest.inc#L696): Remove this use of the "eval" function. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1523)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L32](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/api/ApiQuerySiteinfo.php#L32): Class "ApiQuerySiteinfo" has 29 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L38](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/api/ApiQuerySiteinfo.php#L38): The Cyclomatic Complexity of this function "execute" is 23 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L330](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/api/ApiQuerySiteinfo.php#L330): Remove this unused "$host" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L514](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/api/ApiQuerySiteinfo.php#L514): Do not use the global wgParser Parser instance. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)
1. ![MAJOR][MAJOR] [ApiQuerySiteinfo.php#L522](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/api/ApiQuerySiteinfo.php#L522): Do not use the global wgParser Parser instance. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L15](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/installer/DatabaseInstaller.php#L15): Class "DatabaseInstaller" has 39 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L318](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/installer/DatabaseInstaller.php#L318): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L489](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/installer/DatabaseInstaller.php#L489): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MAJOR][MAJOR] [DatabaseInstaller.php#L513](https://github.com/Wikia/app/blob/b1b21f7ce655159cbc49a770cf462fa8b0c9938f/includes/installer/DatabaseInstaller.php#L513): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 204 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 118 major
* ![MINOR][MINOR] 17 minor
* ![INFO][INFO] 68 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [parserTest.inc#L696](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/tests/parser/parserTest.inc#L696): Remove this use of the "eval" function. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1523)
1. ![MAJOR][MAJOR] [Title.php](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php): File "Title.php" has 4740 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [Title.php#L31](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L31): Class "Title" has 181 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [Title.php#L33](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L33): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L34](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L34): Remove this unused "$titleCache" private field. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)
1. ![MAJOR][MAJOR] [Title.php#L35](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L35): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L62](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L62): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L93](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L93): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L286](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L286): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [Title.php#L288](https://github.com/Wikia/app/blob/7534bbaad89a95074b4063e55b7b425af824f16b/includes/Title.php#L288): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 204 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 118 major
* ![MINOR][MINOR] 17 minor
* ![INFO][INFO] 68 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [parserTest.inc#L696](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/tests/parser/parserTest.inc#L696): Remove this use of the "eval" function. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1523)
1. ![MAJOR][MAJOR] [Title.php](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php): File "Title.php" has 4740 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [Title.php#L31](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L31): Class "Title" has 181 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [Title.php#L33](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L33): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L34](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L34): Remove this unused "$titleCache" private field. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)
1. ![MAJOR][MAJOR] [Title.php#L35](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L35): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L62](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L62): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L93](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L93): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [Title.php#L286](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L286): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [Title.php#L288](https://github.com/Wikia/app/blob/ea9ec7258198db97cf41578d1df92bd1414925b7/includes/Title.php#L288): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: includes/Title.php [hunk 3]
```diff
@@ -755,19 +755,6 @@ public function isTrans() {
 	}
 
 	/**
-	 * Returns the DB name of the distant wiki which owns the object.
```
Comment (ghost) [inline includes/Title.php:758]: I am not so sure about removing core methods.

File: includes/interwiki/Interwiki.php [hunk 4]
```diff
@@ -347,15 +343,6 @@ public function getURL( $title = null ) {
 	}
 
 	/**
-	 * Get the API URL for this wiki
```
Comment (ghost) [inline includes/interwiki/Interwiki.php:350]: Same here, I am not so sure about removing core MW methods.

**Comment by ghost** [comment]
General consideration: aren't we going to wide here? @sqreek , are you sure this will not break the general interwiki functionality?

**Pull request merged**